sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -317,11 +317,6 @@ public partial class OciManifest
         public System.Collections.Generic.IList<Azure.Containers.ContainerRegistry.Specialized.OciBlobDescriptor> Layers { get { throw null; } }
         public int? SchemaVersion { get { throw null; } set { } }
     }
-    public partial class UploadBlobOptions
```

Comment:
I think we use this type in changelog as well.

Reply:
Good call!  I will add it - thank you!